SQLite3/mORMoti18n.pas

```diff
@@ -1907,9 +1931,10 @@ procedure TLanguageFile.Translate(var English: string);
 // case-sensitive (same as standard gettext)
 var result: string;
 begin
-  result := FindMessage(Hash32(
+  //resourcestrings may be not in English

```

Comment:
this comment (and below) is a bit confusing for sure -> need to delete old code, and fix comments

Reply:
removed